data/account-providers/tandem-money-limited.json

```diff
@@ -74,11 +74,11 @@
   "github": null,
   "fca": "001b000000i8ezfAAA",
   "articles": [],
-  "thirdPartyBankingLicense": null,
+  "thirdPartyBankingLicense": false,
```

Comment:
You can't use a boolean for this data point. Should be an empty string "" or null.